bulbs/styles/static/styles/bulbs-styles/components/ads.scss

```diff
@@ -0,0 +1,8 @@
+.dfp-slot-inread[data-ad-load-state="loaded"] {
+  color: #7f7f7f;
```

Comment:
Can we only display 'Content continues below advertisement' if the `data-ad-load-state` is `loaded`?  Otherwise, you see "Content continues below advertisement" and it looks exactly the same as the article body